openmetadata-service/src/main/java/org/openmetadata/service/apps/bundles/changeEvent/Destination.java

```diff
@@ -62,4 +65,36 @@ default void setStatus(
             status, null, attemptTime, statusCode, reason, timestamp, attemptTime);
     getSubscriptionDestination().setStatusDetails(subStatus);
   }
+
+  default void setStatusForTestDestination(
+      TestDestinationStatus.Status status, Integer statusCode, Long timestamp) {
+    TestDestinationStatus subStatus =
+        AlertUtil.buildTestDestinationStatus(status, statusCode, timestamp);
+    getSubscriptionDestination().setStatusDetails(subStatus);
+  }
+
+  default void setStatusForTestDestination(
+      TestDestinationStatus.Status status,
+      Integer statusCode,
+      String statusInfo,
+      Map<String, List<String>> headers,
+      String entity,
+      String mediaType,
+      String location,
+      Long timestamp) {
+
+    TestDestinationStatus subStatus =
```

Comment:
Avoid using two many args , if required build a context object to pass around StatusContext maybe

Reply:
Noted.